### PR TITLE
CAM-11797: docs(rest) add event subscription rest api docs

### PR DESCRIPTION
**Pull request by garima-camunda**
related to https://jira.camunda.com/browse/CAM-11797

### CODE REVIEW

**Comment by garima-camunda** [comment]
related to https://jira.camunda.com/browse/CAM-11797

File: content/reference/rest/event-subscription/get-query.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+GET `/event-subscription?eventType=message&sortBy=created&sortOrder=asc`
+
+## Response
+
+    [{
+     "id":"anId",
+     "eventType":"message",
+     "eventName":"anEventName",
+     "executionId":"anExecutionId",
+     "processInstanceId":aProcessInstanceId,
```
Comment (yanavasileva) [inline content/reference/rest/event-subscription/get-query.md:181]:
```suggestion
     "processInstanceId":"aProcessInstanceId",
```

File: content/reference/rest/event-subscription/get-query.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+     "activityId":"anActivityId",
+     "createdDate":"2020-04-20T15:23:12.229+0200",
+     "tenantId":null
+     },
+     {
+     "id":"anotherId",
+     "eventType":"message",
+     "eventName":"anotherEventName",
+     "executionId":"anotherExecutionId",
+     "processInstanceId":anotherProcessInstanceId,
```
Comment (yanavasileva) [inline content/reference/rest/event-subscription/get-query.md:191]:
```suggestion
     "processInstanceId":"anotherProcessInstanceId",
```

**Comment by yanavasileva** [comment]
As discussed, I squashed, changed the message commit, and merged with (9932281). 🎉

**Pull request closed without merging**